src/app/api/crud/webhooks.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+from typing import List
+
+from app.api import crud
+from app.db import webhooks
+
+
+async def fetch_webhook_urls(route: str) -> List[str]:
```

Comment:
Why not using `fetch_all`?
Because we want to retrieve only the webhook? How about altering fetch_all to get only one field instead?

Reply:
I hesitated but I didn't want to import the webhook table in the alert route code so I had to move it here anyway. Thought I'd go for the most efficient query possible in the process :sweat_smile: